db/migrate/20220810101108_add_default_to_bookings_school_availability.rb

```diff
@@ -0,0 +1,22 @@
+class AddDefaultToBookingsSchoolAvailability < ActiveRecord::Migration[7.0]
+  def up
+    change_column_default :bookings_schools, :availability_info, default_availability_info
+    change_column_default :bookings_schools, :experience_type, :inschool
```

Comment:
Does it still make sense to allow this column to be `null`?

Reply:
Yeah that's a good shout, I'll generate a new migration to enforce not null 👍